src/index.html

```diff
@@ -11,7 +11,215 @@
   <body>
-    <h1>HTML Form</h1>
+    <!--  <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="frame">
+        <legend>
+          Personal information
+        </legend>
+
+        <label class="field">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            autocomplete="off"
+            name="lastName"
+          >
+
+        </label>
+
+        <label class="field">
+          Name:
+          <input
+            type="text"
+            name="firstName"
+            autocomplete="off"
+          >
+
+        </label>
+
+        <label class="field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+
+        </label>
+        <label class="field">
+          Full date of birth:
+          <input
+            type="date"
+            name="years"
+            value="12"
+          >
+
+        </label>
+
+        <label class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+          >
+
+        </label>
+
+      </fieldset>
+
+      <fieldset class="frame">
+        <legend>
+          Registration
+        </legend>
+
+        <label class="field">
+          E-mail:
+          <input
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+            required
+          >
+
+        </label>
+
+        <label class="field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            required
+          >
+
+        </label>
+
+      </fieldset>
+
+      <fieldset class="frame">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <label class="field">
+          Do you love cats?
+
+          <label>
+            <input
+            type="radio"
+            name="answer"
+            value="Yes"
+            >
+              Yes
+
+          </label>
+
+          <label>
+            <input
+            type="radio"
+            name="answer"
+            value="No"
+            >
+              No
+
+          </label>
+
+        </label>
+
+        <label class="field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+
+        </label>
+
+        <label class="field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+          >
+
+        </label>
+
+        <label class="field">
+          What are your favorite brand of cars?
+          <select name="cars"
+          id="cars" multiple>
```

Comment:
Check the code style 
```suggestion
          <select 
            name="cars"
            id="cars" 
            multiple
          >
```